docs/src/run-with-lambda.md

```diff
@@ -0,0 +1,124 @@
+
+Some very brief context: Lambda has two deployment models, zip file and container-based. When using zip file deployment, the online code editor is available, in which we can edit the .yaml configuration. When using container-based deployment, we can pass our configuration on the command line or environment variables.
+
+Everything can be performed from AWS CloudShell, otherwise you will need to install the AWS CLI and the AWS SAM CLI, and configure authentication. The CloudShell also runs in a particular AWS region.
+
+### Container deployment
+
+Lambda images must come from a public or private ECR registry. Pull the image from GHCR and push it to ECR.
+
+```bash
+$ docker pull ghcr.io/maplibre/martin:latest --platform linux/arm64
+$ aws ecr create-repository --repository-name martin
+[…]
+        "repositoryUri": "493749042871.dkr.ecr.us-east-2.amazonaws.com/martin",
+# Read the repositoryUri which includes your account number
+$ docker tag ghcr.io/maplibre/martin:latest 493749042871.dkr.ecr.us-east-2.amazonaws.com/martin:latest
+$ aws ecr get-login-password --region us-east-2 | docker login --username AWS --password-stdin 493749042871.dkr.ecr.us-east-2.amazonaws.com
+$ docker push 493749042871.dkr.ecr.us-east-2.amazonaws.com/martin:latest
+```
+
+Now you can go to the [Lambda console](https://console.aws.amazon.com/lambda) and create your function.
+
+1. Click “Create function”.
+2. Choose “Container image”.
+3. Put something in “Function name”. (Note: This is an internal identifier, not exposed in the function URL.)
+4. Click “Browse images”, and select your repository and the tag. (If you can’t find it, see if you’re in the same region?)
+5. Expand “Container image overrides”, and under CMD put the URL of a .pmtiles file.
+6. Set “Architecture” to arm64 to match the platform that we pulled. (Lambda has better ARM CPUs than x86.)
+7. Click “Create function”.
+8. Find the “Configuration” tab, select “Function URL”, “Create function URL”.
+9. Set “Auth type” to NONE. (Don’t enable CORS, as Martin already will, and you will have duplicate headers and your CORS will be broken.)
+10. Click on the “Function URL”. If it works, hooray! If it doesn’t, open the “Monitor” tab, “View CloudWatch logs”, find the most recent Log stream.
+
+### Zip deployment
+
+It’s possible to deploy the entire codebase from the AWS console, but we will use Serverless Application Model. Our function will consist of a “Layer”, containing the Martin binary, and our function itself will contain the configuration in .yaml format.
+
+#### The layer
+
+Download the binary and place it in your staging directory. The `bin` directory of your Layer will be added to the PATH.
+
+```bash
+mkdir -p martin_layer/src/bin/
+cd martin_layer
+curl -OL https://github.com/maplibre/martin/releases/download/VERSION_NUMBER_HERE/martin-aarch64-unknown-linux-musl.tar.gz
```

Comment:
Use `latest` instead of requiring version number

```suggestion
curl -OL https://github.com/maplibre/martin/releases/latest/download/martin-aarch64-unknown-linux-musl.tar.gz
```